### PR TITLE
Hetzner SSH module

### PR DESCRIPTION
**Pull request by timetinytim**
In an effort to make sure as much of our infrastructure is managed automatically as possible, I quickly created a module for handling SSH public keys.

The general goal here being: as much of our infrastructure should be in source control as possible.

Let me know if this goal makes sense!

### CODE REVIEW

File: output.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  value       = hcloud_ssh_key.admin_keys.*.id
+}
+
+output "user_key_ids" {
+  description = "Hetzner IDs for user SSH keys"
+  value       = hcloud_ssh_key.user_keys.*.id
+}
+
+output "admin_key_fingerprints" {
+  description = "Hetzner IDs for admin SSH keys"
```
Comment (renchap) [inline output.tf]: This probably needs to be updated

File: output.tf [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  value       = hcloud_ssh_key.admin_keys.*.fingerprint
+}
+
+output "user_key_fingerprints" {
+  description = "Hetzner IDs for user SSH keys"
```
Comment (renchap) [inline output.tf]: This probably needs to be updated

File: output.tf (hunk 1 above)
Comment (timetinytim) [reply output.tf]: Ah good catch! My bad

**Review by renchap — APPROVED** [review]
👍

**Pull request merged**